lib/rubocop/cop/rspec/capybara/feature_methods.rb

```diff
@@ -45,13 +45,15 @@ class FeatureMethods < Cop
             feature:    :describe
           }.freeze
 
-          def_node_matcher :feature_method?, <<-PATTERN
-            (send {(const nil :RSpec) nil} ${:#{MAP.keys.join(' :')}} ...)
+          def_node_matcher :feature_method, <<-PATTERN
+            (block
+              $(send {(const nil :RSpec) nil} ${#{MAP.keys.map(&:inspect).join}} ...)
```

Comment:
I'm surprised `.join`, rather than `.join(' ')`, is still passing.

Reply:
Probably the pattern is split on every `:` in a string. I would prefer if you keep the `(' ')` though :-)